### ISSUE TITLE
Expose extra_attribute/extra_fields in Vandal UI

### ISSUE THREAD

**Issue by jasonkarns** [graphiti-api/vandal#11]
After adding an `extra_attribute` in a Graphiti Resource, I was expecting to see the attribute listed in Vandal such that I could use the vandal UI to opt-in to including the extra field (by adding `extra_fields` to the request).

I assume Vandal isn't already expected to recognize the `extra_attribute` fields in the UI, so this would be a feature request, not a bug.

Opening this issue to confirm that it's not currently supported (and to serve as a note to others who search for it).

**Comment by jasonkarns** [comment]
should this have been in https://github.com/graphiti-api/vandal_ui ?

**Comment by richmolj** [comment]
Yeah, we need to add support for extra attrs to Vandal, good issue. I think this repo makes sense (frontend code, vandal_ui is largely the engine to host that code)
